UPGRADE-6.3.md

```diff
@@ -220,6 +220,103 @@ German date format. For backwards compatibility reasons they work with German da
 - Changed method `WriteLogEntry()`: parameter `$sLogFileName` is now ignored.
 - Changed method `WriteLogEntrySimple()`: parameter `$sLogFileName` is now ignored.
 
+## Backend Theme Library
+
+The Backend was upgraded to Bootstrap 4.1.3.
+
+See the [Bootstrap Migration Guide](https://getbootstrap.com/docs/4.1/migration/) for needed changes to your backend modules.
+
+During the upgrade to Bootstrap 4 the following styles where checked and these are our findings:
+
+.img-responsive -> .img-fluid
+- TCMSFieldMedia
+- TCSMFieldFMapCoordinate
+
+btn-default -> btn-secondary
+- Some TCMSField types and TCMSTableEditors so check yours.
+
+.pull-left -> .float-left
+- TCMSFieldDocument
+- TCMSFieldDocumentProperties
+- TCMSFieldMediaProperties
+- TCMSFieldModuleInstance
+- MTHeader
+- header navigation
+- footer
+- Added CSS class migration in TCMSRender::DrawButton method for backwards compatibility
+
+.input-sm -> .form-control-sm
+- Almost all TCMSField classes and Twig templates
+- Some list managers
+- Some TCMSTableEditor classes
+
+.table-condensed -> .table-sm
+- TCMSFieldDocumentProperties
+- TCMSFieldMediaProperties
+- TFullGroupTable
+
+New: .page-item + .page-link
+- Pagination in TFullGroupTable
+
+.pull-rigt -> .float-right
+- TFullGroupTable
+- MTHeader
+- MTTableditor
+
+.input-group-addon -> .input-group-append
+- Field types using the text length counter addon (varchar)
+
+.navbar-default -> .navbar-light
+- TCMSTableManager and the layout manager iframes
+
+.navbar-toggle -> .navbar-toggler
+- header navbar
+
+.col-md-* -> .col-lg-*
+- header
+- login
+
+Not found anywhere (so you might want to skip this search, too):
+- well
+- thumbnail
+- list-line
+- page-header
+- dl-horizontal
+- blockquote
+- btn-xs
+- btn-group-justified
+- btn-group-xs
+- breadcrumb
+- center-block
+- img-responsive
+- img-rounded
+- form-horizontal
+- radio
+- checkbox
+- input-lg
+- control-label
+- hidden/visible-xs, sm, md, lg
+- label
+- navbar-form
+- navbar-btn
+- progress-bar*
+
+## Backend JQuery
+
+JQuery that is used in the Chameleon backend was upgraded to version 3.3.1. For backwards compatibility
+jquery.migrate 1.4.1 was added, but will be removed in a future Chameleon version.
```

Comment:
But all tasks printed there are done for the backend?

Reply:
This is to be read in the context of the whole document, so we need to make clear for every section that it only affects the backend.